src/Common/src/Interop/Linux/procfs/Interop.ProcFsStat.cs

```diff
@@ -67,16 +72,104 @@ internal struct ParsedStat
             //internal long cguest_time;
         }
```

Comment:
Ah nevermind...this looks to be some tickery in the GitHub code review tool...pay no attention :)